### ISSUE TITLE
Ablaufdatum von Bild nicht ganz so weit in die Zukunft setzen.

### ISSUE THREAD

**Issue by Carpetcrawlwer** [rincewind/kioskbackend#16]
Zurück scrollen ist auf dem Handy aus dem Jahr 3000 mega anstrengend

**Comment by rincewind** [comment]
Ja. Den Datepicker habe ich heute noch schnell eingebaut und dann mit Zarah festegestellt, dass das Kacke ist. Und sie auch lieber eine direkte Eingabemöglichkeit möchte.
